sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/connection/ConnectionProvider.scala

```diff
@@ -46,8 +47,10 @@ private[jdbc] object ConnectionProvider extends Logging {
           logError(s"Failed to load built in provider.", t)
       }
     }
-    // Seems duplicate but it's needed for Scala 2.13
-    providers.toSeq
+
+    val disabledProviders = SQLConf.get.disabledJdbcConnectionProviders.split(",")
```

Comment:
Looks like there's a util for this `Utils.stringToSeq`.

Reply:
Changed, however I've taken this solution from existing params. After this I'm going to file a PR to change them.